### ISSUE TITLE
[time.duration.cons] sentence in p3 is not clear

### ISSUE THREAD

**Issue by cpplearner** [cplusplus/draft#4476]
[\[time.duration.cons\]/3](http://eel.is/c++draft/time.duration.cons#3):

> Constraints: No overflow is induced in the conversion and `treat_­as_­floating_­point_­v<rep>` is true or both `ratio_­divide<Period2, period>​::​den` is `1` and `treat_­as_­floating_­point_­v<Rep2>` is `false`.

1. There are a bit too many conjunctions in this sentence, making the sentence hard to parse.
2. It's unclear what "No overflow is induced in the conversion" means. Based on [LWG 2094](https://cplusplus.github.io/LWG/issue2094), I guess it means that no overflow shall occur when computing `ratio_divide`, but this is not obvious.
   (My initial reading is that `duration<int, micro> us = duration<int, milli>::max();` should be an error due to overflow, but the [example](http://eel.is/c++draft/time.duration.cons#example-2) shows that `ms` _is_ convertible to `duration<int, micro>`, and we can't SFINAE on argument value).

**Comment by johelegp** [comment]
https://wg21.link/LWG3090 also touches on 2.

**Comment by jensmaurer** [comment]
Adding a comma before "or both" probably addresses your first concern.

**Comment by jwakely** [comment]
> https://wg21.link/LWG3090 also touches on 2.

My proposed resolution also resolves 1. I think this is an exact dup of LWG 3090.
